sympy/controlsys/lti.py

```diff
@@ -0,0 +1,757 @@
+        lcd_coeff = Poly(lcd, s).all_coeffs()[1:]
+
+        # get the degree of the lcd
+        lcd_deg = degree(lcd, s)
+
+        # get the Matrix Valued Coeffs of G_sp in G_sp = 1/lcd * (N_1 * s**(n-1) + N_2 * s**(n-2) .. +N_n)
+        G_sp_coeff = _matrix_coeff(simplify(G_sp * lcd), s)
+        G_sp_coeff = [zeros(m, k)] * (lcd_deg - len(G_sp_coeff)) + G_sp_coeff
+
+        # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+        # now store A, B, C, D in terms of the coefficients of lcd and G_sp
+        # define A
+        A = (-1) * lcd_coeff[0] * eye(k)
+
+        for alpha in lcd_coeff[1:]:
+            A = A.row_join((-1) * alpha * eye(k))
+
+        for i in xrange(lcd_deg - 1):
+            if i == 0:
+                tmp = eye(k)
+            else:
+                tmp = zeros(k)
+
+            for j in range(lcd_deg)[1:]:
+                if j == i:
+                    tmp = tmp.row_join(eye(k))
+                else:
+                    tmp = tmp.row_join(zeros(k))
+            if tmp is not None:
+                A = A.col_join(tmp)
+
+        # define B
+        B = eye(k)
+        for i in xrange(lcd_deg - 1):
+            B = B.col_join(zeros(k))
+
+        # define C
+        C = G_sp_coeff[0]
+        for i in range(lcd_deg)[1:]:
+            C = C.row_join(G_sp_coeff[i])
+
+        # return the state space representation
+        return [simplify(A), simplify(B), simplify(C), simplify(D)]
+
+    #
+    # evaluate(self, u, t)
+    #
+    def evaluate(self, u, x0, t, t0=0, method=None, return_pretty=False, do_integrals=True, dps=5):
+        """evaluate the system output for an input u
+
+        The output of the system y for the output u if given by solving the state equation for x
+        and than substituting that into the output equation
+
+        Parameters
+        ==========
+
+        u  : one-column matrix
+            The input vector in time-space
+        x0 : one-column matrix
+            the state of the system at time t0
+        t  : symbol, tuple (t,[list of times])
+            if t is only a symbol, the system is evaluated simbolycaly.
+            if t is a tuple of a symbol and a list, the symstem is evaluated numericaly, at the given times in the list
+        t0 = 0 : number
+            the time t0 at which the state of the system is known
+
+        method : Bool
+            not supported yet, always uses diagonalizaton
+        return_pretty : Bool
+            if True, the funtion returns a tuple of equations showing the input, initial conditions, and the output
+        do_integrals : Bool
+            if True, the function tries to evaluate the integrals in the solution. if False, it returns an
+            Integral object instead. Only valid for symbolic solutions, ignored otherwise
+        dps : integer
+            the decimal precision of numericial integration
+
```

Comment:
It would be useful (and typical) to include and "Examples" section in each of these docstrings to show how the class is used. Check out other SymPy classes.